xicam/core/intents/__init__.py

```diff
@@ -55,3 +55,18 @@ def canvas_name(self):
             y_name = self.labels.get("left", "")
             return x_name + ", " + y_name
         return self._canvas_name
+
+
+class ErrorBarIntent(Intent):
```

Comment:
It may be beneficial to describe that `ErrorBarIntent` represents a plot with error bars?

Reply:
Should be clearer now by inheritance